lego/settings/test.py

```diff
@@ -21,6 +21,7 @@
 SERVER_URL = "http://127.0.0.1:8000"
 FRONTEND_URL = "http://127.0.0.1:8000"
 SERVER_EMAIL = "Abakus <no-reply@abakus.no>"
+ENVIRONMENT_NAME = "testing"
```

Comment:
This is needed because it crashes with running tests because `program.ENVIRONMENT_NAME` is not set. Also, it does not affect anything else having it there.